libs/wingc/src/type_check.rs

```diff
@@ -1084,6 +1070,18 @@ impl TypeRef {
 		return false;
 	}
 
+	pub fn is_preflight_type(&self) -> bool {
```

Comment:
`is_preflight_type` sounds like it might include preflight functions. Maybe `is_preflight_object` would describe it better?

Reply:
changed to `is_preflight_object_type`